src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -615,6 +654,8 @@ type WebAppBuilder() =
     /// Automatically add the ASP.NET Core logging extension.
     [<CustomOperation "automatic_logging_extension">]
     member _.DefaultLogging (state:WebAppConfig, setting) = { state with AutomaticLoggingExtension = setting }
+    [<CustomOperation "custom_domain">]
+    member _.CustomDomain(state:WebAppConfig, customDomain) = { state with CustomDomain = customDomain }
```

Comment:
It would be useful to have some additional overloads here.
```
    member _.CustomDomain(state:WebAppConfig, customDomain) = { state with CustomDomain = InsecureDomain customDomain }
    member _.CustomDomain(state:WebAppConfig, (customDomain,sslState)) = { state with CustomDomain = SecuredDomain(customDomain,sslState)}